spec/mailer/planning_application_mailer_spec.rb

```diff
@@ -4,27 +4,38 @@
 
 RSpec.describe PlanningApplicationMailer, type: :mailer do
   let(:local_authority) do
-    create :local_authority,
-           name: "Cookie authority",
-           subdomain: "cookies",
-           signatory_name: "Mr. Biscuit",
-           signatory_job_title: "Lord of BiscuitTown",
-           enquiries_paragraph: "reach us on postcode SW50",
-           email_address: "biscuit@somuchbiscuit.com"
+    create(
+      :local_authority,
+      name: "Cookie authority",
+      subdomain: "cookies",
+      signatory_name: "Mr. Biscuit",
+      signatory_job_title: "Lord of BiscuitTown",
```

Comment:
🤷 😆

Reply:
haha you'll find some random names like this throughout, thanks to Melissa